### DISCUSSION TITLE
Which SMPP should i use?

### DISCUSSION THREAD

**Discussion by halim-kahraman** [jookies/jasmin#1068] [General]
Hey to everyone, i've tried to install jasmin on a raspberry pi bullseye, everything fine!

Should i subscribe a provider on the internet to send SMS to any device, or am i creating my own SMPP Server which emulates and works like an operator SMS-C? 

My goal is to send via HTTP or SMPP to a group SMS with my own SENDER-ID for example "GroupGit" as SENDER-ID and Not with the mobile number of any sim-card.

Actually i am using playsms with gammu and an mobile hsdpa stick with a simcard. Everything works Fine, but the SENDER-ID is always the phone-number of the SIM which is installed in the HSDPA USB Dongle.

Is there a way to solve this Problem with the own created SMPP via jasmind?

Thanks in advance.

**Comment by farirat** [comment]
Jasmin provides smpp server, smpp client and http api communication channels, it does not provide usb/serial channel with hardware modems.

**Reply by halim-kahraman** [reply]
Hey Farirat,

thx for the quick reply.
Ok then, how can i send the sms when i don't have a sim-Card? 

Am i using my own SMSC then? Does my own SMSC send the message via SMPP Protocol to the Provider and do they charge me or is it free, becaus they transmit it via the Internet?

Thx in advice.

Halim

**Comment by tarikogut** [comment]
halim you have to install jasmin and buy sms credits from provider

halim-kahraman ***@***.***>, 13 Ağu 2022 Cmt, 03:24 tarihinde
şunu yazdı:

**Comment by halim-kahraman** [comment]
Hey Tarik, thx a Lot, which provider would you recommend me?
